vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -35,6 +35,11 @@
 #include "base/strings/utf_string_conversions.h"
 #include "base/time/time.h"
 
+#if defined(OS_ANDROID)
+#include "base/system/sys_info.h"
```

Comment:
We do not require a guard here for Android as part of base and works cross platform

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (16)